src/main/java/io/stargate/sgv2/jsonapi/service/operation/model/impl/CreateCollectionOperation.java

```diff
@@ -59,17 +63,22 @@ public static CreateCollectionOperation withVectorSearch(
 
   public static CreateCollectionOperation withoutVectorSearch(
       CommandContext commandContext,
+      DatabaseLimitsConfig dbLimitsConfig,
       ObjectMapper objectMapper,
       SchemaManager schemaManager,
       String name) {
     return new CreateCollectionOperation(
-        commandContext, objectMapper, schemaManager, name, false, 0, null, null);
+        commandContext, dbLimitsConfig, objectMapper, schemaManager, name, false, 0, null, null);
   }
 
   @Override
   public Uni<Supplier<CommandResult>> execute(QueryExecutor queryExecutor) {
     return schemaManager
-        .getTable(commandContext.namespace(), name, MISSING_KEYSPACE_FUNCTION)
+        .getTables(commandContext.namespace(), MISSING_KEYSPACE_FUNCTION)
+        .collect()
+        .asList()
+        .invoke(tables -> verifyLimits(tables))
```

Comment:
If a user rerun the same createCollection command which is already created, this logic will throw error  ErrorCode.TOO_MANY_COLLECTIONS instead of duplicate error. Is this the expectation?

Reply:
Good point; I was wondering about that. Probably should only throw exception if new would be created -- will change, test that case.

Reply:
@maheshrajamani Fixed logic so that max-collections only checked if table does not yet exist; added verification of this case in test too.